radix-engine/src/vm/wasm/constants.rs

```diff
@@ -15,6 +15,7 @@ pub const ADDRESS_GET_RESERVATION_ADDRESS_FUNCTION_NAME: &str = "address_get_res
 pub const OBJECT_NEW_FUNCTION_NAME: &str = "object_new";
 pub const OBJECT_GLOBALIZE_FUNCTION_NAME: &str = "object_globalize";
 pub const OBJECT_INSTANCE_OF_FUNCTION_NAME: &str = "object_instance_of";
+pub const OBJECT_BLUEPRINT_ID_FUNCTION_NAME: &str = "object_blueprint_id";
```

Comment:
Can we make this a verb like the others, ie "object_get_blueprint_id"

Reply:
Yes!